lib_xua/src/core/main.xc

```diff
@@ -534,6 +538,8 @@ int main()
 #if (NUM_USB_CHAN_OUT > 0) || (NUM_USB_CHAN_IN > 0) || XUA_HID_ENABLED || defined(MIDI)
             /* Core USB audio task, buffering, USB etc */
             {
+                XUA_USER_FUNCTION_CALL_PRE_BUFFER
```

Comment:
For function calls, weak symbols are probably nicer? Or so do need to pass params (potentially then requiring var args?)

Reply:
No I wouldn't recommend using va, just it its a requirement then the weak symbols are not a goer.